source/sass/components/article-page.scss

```diff
@@ -345,7 +345,7 @@
     font-weight: 600;
 
     .rich-text * {
-      color: $white !important;
+      color: $white;
```

Comment:
no need to have `!important` here

Reply:
content in header cells should appear white

test page: https://foundation-s-issue-6355-xnjp28.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/
<img width="967" alt="image" src="https://user-images.githubusercontent.com/2896608/112395580-e98c6f80-8cbb-11eb-89ee-fe81696e2253.png">